webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const webpack = require('webpack');
```

Comment:
If your starting a new react project I'd look into Vite (https://vitejs.dev/) or Next.js (https://nextjs.org/)

Reply:
Its a lot faster than webpack and less config to setup. I am guessing you used Create react app?

https://react.dev/learn

Reply:
> If your starting a new react project I'd look into Vite (https://vitejs.dev/) or Next.js (https://nextjs.org/)

https://nextjs.org/ seems to be pretty cool and automate the whole process. Almost no setup is required and a lot of magic happening which is hard to understand how all the automation is happening. I will build new project using nextjs as migrating existing one will take more efforts